documentation/staging/content/userguide/managing-domains/domain-events.md

```diff
@@ -285,3 +288,100 @@ Source:
 Type:    Normal
 Events:  <none>
 ```
+
+Example of the sequence of operator generated events in a domain rolling restart after the domain resource's `image` and `logHomeEnabled` changed.
+
+The output of command `kubectl get events -n sample-domain1-ns --selector=weblogic.domainUID=sample-domain1,weblogic.createdByOperator=true --sort-by=lastTimestamp'.
+
+```
+LAST SEEN   TYPE     REASON                     OBJECT           MESSAGE
+2m58s       Normal   DomainChanged              domain/sample-domain1   Domain resource sample-domain1 was changed
+2m58s       Normal   DomainProcessingStarting   domain/sample-domain1   Creating or updating Kubernetes presence for WebLogic Domain with UID sample-domain1
+2m58s       Normal   DomainRollStarting         domain/sample-domain1   Rolling restart WebLogic server pods in domain sample-domain1 because: 'image' changed from 'oracle/weblogic' to 'oracle/weblogic:14.1.1.0',
+  'logHome' changed from 'null' to '/shared/logs/sample-domain1'
+2m58s       Normal   PodCycleStarting           domain/sample-domain1   Replacing pod sample-domain1-adminserver because: In container 'weblogic-server':
+  'image' changed from 'oracle/weblogic' to 'oracle/weblogic:14.1.1.0',
+  env 'LOG_HOME' changed from 'null' to '/shared/logs/sample-domain1'
```

Comment:
If this is easy to fix, then it'd be more clear to report that logHome changed -- env 'LOG_HOME' was not directly changed by user, the operator changed it because logHome changed...

Reply:
This is the POD cycle event where all we have is the pod spec, and therefore all fields are reported as what they are in the spec. We could change this particular env var to the corresponding domain resource field, but it would not be consistent with other env vars changes possibly reported in the same event.

Reply:
Ok.